pkg/api/types.go

```diff
@@ -76,7 +76,7 @@ type FeatureFlags struct {
 	EnableTelemetry           bool `json:"enableTelemetry,omitempty"`
 	EnableIPv6Only            bool `json:"enableIPv6Only,omitempty"`
 	EnableWinDSR              bool `json:"enableWinDSR,omitempty"`
-	EnforceUbuntu2004DisaStig bool `json:"enforceUbuntu2004DisaStig,omitempty"`
+	EnforceUbuntu2204DisaStig bool `json:"enforceUbuntu2204DisaStig,omitempty"`
```

Comment:
this is a breaking change, update will likely break, we have to add a new property.